egui/src/containers/collapsing_header.rs

```diff
@@ -260,6 +285,15 @@ impl CollapsingHeader {
             });
         }
 
+        let selected = self.selected || header_response.hovered() || header_response.has_focus();
+        if self.selectable && selected {
```

Comment:
From the docstring of `selected` it seems like we should show it as selected even if it is not selectable. This can be useful if the selection is done via some other means (e.g. as a highlight).

```suggestion
        if self.selected || self.selectable && (header_response.hovered() || header_response.has_focus()) {
        }
```